src/components/modals/AddAccounts/steps/03-step-import.js

```diff
@@ -85,11 +85,13 @@ const Desc = styled(Box).attrs({
 `
 
 const SectionAccounts = ({ defaultSelected, ...rest }: *) => {
+  // componentDidMount-like effect
   useEffect(() => {
     if (defaultSelected && rest.onSelectAll) {
       rest.onSelectAll(rest.accounts)
     }
-  })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```

Comment:
nice one \o/

Reply:
`git bisect` is :ok_hand: at tracking bugs